### PR TITLE
feat(components): [color-picker] add `show` `hide` expose

### LINKED ISSUE #11926: [Component] [color-picker] color-picker 由外界触发打开
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.0`
- Element Plus Version: `2.2.26`
- Browser / OS: `chrome`
- Build Tool: `Webpack`

### Related Component

- `el-color-picker`

### Reproduction Link

[Github Repo](https://github.com/element-plus/element-plus/blob/dev/packages/components/color-picker/src/color-picker.vue)

### Steps to reproduce

更新后无法调用到内部的 handleTrigger 打开选择器了，看了代码改为了只暴露 color

### What is Expected?

暴露方法支持外部打开 color-picker
或 暴露 handleTrigger

### What is actually happening?

仅暴露color

### Additional comments

_(empty)_

### PR DESCRIPTION
**Pull request by xiterjia**
- Add `handleTrigger` expose

closed #11926

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by chenxch** [comment]
@xiterjia Good job, would you mind exposing hide as well?

**Comment by xiterjia** [comment]
> @xiterjia Good job, would you mind exposing hide as well?

Yeah, expose `show` and `hide` functions to replaced `handleTrigger`

**Review by ryuhangyeong — APPROVED** [review]

File: packages/components/color-picker/src/color-picker.vue [hunk 1]
```diff
@@ -217,6 +217,11 @@ function setShowPicker(value: boolean) {
 
 const debounceSetShowPicker = debounce(setShowPicker, 100)
 
+function show() {
+  if (colorDisabled.value) return
+  debounceSetShowPicker(true)
```
Comment (chenxch) [inline packages/components/color-picker/src/color-picker.vue:222]: Manual mode may not require debounce processing.

Comment (xiterjia) [reply packages/components/color-picker/src/color-picker.vue]: Use `setShowPicker`.

**Review by chenxch — APPROVED** [review]

**Pull request merged**